ocs_ci/templates/app-pods/warp.yaml

```diff
@@ -0,0 +1,32 @@
+kind: DeploymentConfig
+apiVersion: apps.openshift.io/v1
+metadata:
+  name: warppod
+  labels:
+    app: warppod
+spec:
+  template:
+    metadata:
+      labels:
+        app: warppod
+    spec:
+      securityContext:
+        fsGroup: 2000
+      serviceAccountName: admin
+      restartPolicy: Always
+      volumes:
+      - name: warp-vol
+        persistentVolumeClaim:
+          claimName: warp-s3
+      containers:
+      - name: warp
+        image: quay.io/tiffanyn108/warp:latest
+        resources:
+          limits:
+            memory: "2048Mi"
```

Comment:
I see that you are running one instance if the pod. Is there a reason to limit pod resource?

Reply:
I only need a small fedora pod to run the test.  No specific reason.